src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--mon calendar__month--31">
```

Comment:
when I change class to` calendar__start-day--tue calendar__month--29`, first day in calendar should be tuesday, and there should be 29 days in the calendar. It doesn't work this way now